.github/workflows/reusable-create-java-contrib-pull-request.yml

```diff
@@ -0,0 +1,67 @@
+name: Reusable - Create Java contrib pull request
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        type: string
+        required: true
+    secrets:
+      BOT_TOKEN:
+        required: true
+  # to help with partial release build failures
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Version"
+        required: true
+
+jobs:
+  create-java-contrib-pull-request:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: opentelemetrybot/opentelemetry-java-contrib
+          # this is the personal access token used for "git push" below
+          token: ${{ secrets.BOT_TOKEN }}
+
+      - name: Initialize pull request branch
+        env:
+          VERSION: ${{ inputs.version }}
+        run: |
+          git remote add upstream https://github.com/open-telemetry/opentelemetry-java-contrib.git
+          git fetch upstream
+          git checkout -b update-opentelemetry-sdk-to-${VERSION} upstream/main
+
+      - name: Update version
+        env:
+          VERSION: ${{ inputs.version }}
+        run: |
+          sed -i "s/^val otelVersion = .*/val otelVersion = \"$VERSION\"/" dependencyManagement/build.gradle.kts
+
+      - name: Use CLA approved github bot
+        run: |
+          # cannot run the use-cla-approved-github-bot.sh script here since in a different repo
+          git config user.name opentelemetrybot
+          git config user.email 107717825+opentelemetrybot@users.noreply.github.com
```

Comment:
Contrib has that script, but I can perfectly understand why you'd prefer not to use it here 👍

Reply:
oh, right 😄